.github/workflows/release.yml

```diff
@@ -0,0 +1,78 @@
+name: Stable release 🕊️📦
+
+on:
+  release:
+    types:
+      - released
+      - prereleased
+
+jobs:
+  docker:
+    name: Build Docker images 💿💻
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout ⬇️
+        uses: actions/checkout@v2.4.0
+
+      - name: Setup node environment ⚙️
+        uses: actions/setup-node@v2.5.0
+        with:
+          node-version: 16
+          cache: 'npm'
+          check-latest: true
+
+      - name: Install dependencies 📦
+        run: npm ci --no-audit
+
+      - name: Build client 🛠️
+        run: npm run build
+
+      - name: Configure QEMU ⚙️
+        uses: docker/setup-qemu-action@v1.2.0
+
+      - name: Configure Docker Buildx ⚙️
```

Comment:
Buildx or BuildX? Honestly not sure.

Reply:
Buildx it seems https://docs.docker.com/buildx/working-with-buildx/